src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java

```diff
@@ -44,6 +47,53 @@ private ScannedDocumentsHelper() {
             .collect(toList());
     }
 
+    public static void setExceptionRecordIdToScannedDocuments(
+        ExceptionRecord exceptionRecord,
+        CaseUpdateDetails caseDetails
+    ) {
+        @SuppressWarnings("unchecked")
+        var caseData = (Map<String, Object>) caseDetails.caseData;
+        List<ScannedDocument> scannedDocuments = getScannedDocuments(caseData);
+
+        List<String> exceptionRecordDcns = exceptionRecord.scannedDocuments
+            .stream()
+            .map(scannedDocument -> scannedDocument.controlNumber)
+            .collect(toList());
+
+        List<ScannedDocument> updatedScannedDocuments = scannedDocuments.stream()
+            .map(scannedDocument -> {
+                if (exceptionRecordDcns.contains(scannedDocument.controlNumber)) {
```

Comment:
`Outstanding question: if the case already has scanned documents from other exception records with exceptionRecordReference fields set - should it be responsibility of the service to preserve these fields or should orchestrator restore them not relying on the service?
The answer is - it should be responsibility of the service; this has been checked with Ademola Adeoti`

this overrides even it has already exceptionRecordReference field set.